File: Easy/even_numbers.py
data = ['701', '4123', '2936']

#import sys
#test_cases = open(sys.argv[1], 'r')
#data = test_cases.readlines()
for test in data:
	test = test.rstrip()
	num = int(test)
	if num % 2 == 0:
		print 1
	else:
		print 0
	

#test_cases.close()

File: Easy/even_numbers.r
test.cases <- c("701", "4123", "2936")

#args <- commandArgs(trailingOnly=TRUE)
#test.cases <- strsplit(readLines(args[[1]], warn=FALSE), '\n')
cat(sapply(test.cases, function(test) {
	if (length(test) > 0) {
		num <- as.numeric(test)
		even <- 0
		if (num %% 2 == 0){
			even <- 1
		}
		even
	}
}), sep = "\n")